### PR TITLE
Schemas: Fix enum values not being selectable

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Fixes a missing emit that was causing values to not be able to be selected

### CODE REVIEW

File: src/schemas/components/SchemaFormPropertyString.vue [hunk 1]
```diff
@@ -60,13 +60,13 @@
...
     }
-    
+
     if (stringEnum) {
       return withProps(PCombobox, {
         modelValue: props.value,
         state: props.state,
         options: stringEnum.filter(isString),
-        'onUpdate:modelValue': (value) => asType(value, String),
+        'onUpdate:modelValue': (value) => update(asType(value, String)),
```
Comment (znicholasbrown) [inline src/schemas/components/SchemaFormPropertyString.vue:69]: Enums can have types that aren't strings - is this still correct?

Comment (pleek91) [reply src/schemas/components/SchemaFormPropertyString.vue:69]: hmm this is specifically for properties of type `string`. So I think this is correct? But that probably means I need to account for enums for other things. Numbers and integers probably? What else could an enum be? A boolean?

Comment (pleek91) [reply src/schemas/components/SchemaFormPropertyString.vue:69]: Add added support for integer enums in https://github.com/PrefectHQ/prefect-ui-library/pull/2114/commits/3eeca3f158e701b1607211a14f68743d3269bf05

Comment (znicholasbrown) [reply src/schemas/components/SchemaFormPropertyString.vue:69]: Hm ok I think we might need to expand support there but this is a strict improvement so going to approve

**Review by znicholasbrown — APPROVED** [review]

File: src/schemas/components/SchemaFormPropertyString.vue (hunk 1 above)
Comment (pleek91) [reply src/schemas/components/SchemaFormPropertyString.vue:69]: Love to work with you on that if you have some time. Let me know!

**Pull request merged**